pkg/operator/starter.go

```diff
@@ -120,6 +127,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	klog.Info("Starting controllerset")
 	go csiControllerSet.Run(ctx, 1)
 	go caSyncController.Run(ctx, 1)
+	go configObserver.Run(ctx, 1)
```

Comment:
This is the only file that should be changed in the operator. The rest of the changes in this PR will be moved to library-go.